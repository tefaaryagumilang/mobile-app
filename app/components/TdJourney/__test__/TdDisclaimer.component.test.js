import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';
import TdDisclaimer from '../TdDisclaimer.component';

const spy = jest.fn();
const store = createStore(() => ({}));
const DecoratedForm = reduxForm({
  form: 'testForm',
  onSubmit: spy
}
)(TdDisclaimer);

describe('TdForm should render', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DecoratedForm />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
