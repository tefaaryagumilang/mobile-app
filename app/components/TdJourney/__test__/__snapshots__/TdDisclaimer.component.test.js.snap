// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TdForm should render renders correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "alignItems": "stretch",
      "borderColor": "#ccc",
      "borderWidth": 1,
      "flexGrow": 1,
    }
  }
  keyboardShouldPersistTaps="handled"
>
  <View>
    <View
      style={
        Object {
          "flex": 1,
          "marginHorizontal": 20,
          "marginTop": 20,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 24,
            },
            Object {
              "marginVertical": 10,
            },
          ]
        }
      >
        Syarat dan ketentuan Deposito berjangka
      </Text>
      <View
        style={
          Object {
            "padding": 10,
          }
        }
      />
    </View>
  </View>
</RCTScrollView>
`;
